yaks/README.md

```diff
@@ -552,6 +552,28 @@ yaks = Yaks.new do
 end
 ```
 
+Yaks also has support for respecting the `include` query parameter (e.g.
+`include=posts.comments,account`), which is a behaviour you can include in
+your mappers:
+
+```ruby
+require "yaks/behaviour/optional_includes"
+
+class UserMapper < Yaks::Mapper
+  include Yaks::Behaviour::OptionalIncludes
+
+  has_many :posts
+  has_one :account
+end
+
+# ...
+
```

Comment:
To fix ataru, try adding this in `ataru_setup.rb` at top level:
```ruby
User = Struct.new
```
this in the `Setup` module:
```ruby
def user
  User.new
end
```
and this in the README here:
```ruby
yaks = Yaks.new
```
and run ataru locally with `ataru check`.